discovery-provider/src/queries/notifications.py

```diff
@@ -1164,10 +1165,30 @@ def solana_notifications():
                 }
             )
 
+        reaction_results: List[Reaction] = (
+            session.query(Reaction)
+            .filter(Reaction.slot >= min_slot_number, Reaction.slot <= max_slot_number)
+            .all()
+        )
```

Comment:
just curious, how come we don't .map this in the final step here?

Reply:
this .filter is a sqlalchemy .filter, not a python .filter :)

Reply:
ah word, so the .all doesnt turn it into a list that can be mapped?

Reply:
it does, but map is kinda clunky in python - it's `map(list, func)` rather than `list.map(func)` which makes the chaining fairly illegible imo

Reply:
ahh gotcha, yeah makes sense, thanks!